src/components/DateRangePicker.jsx

```diff
@@ -125,9 +136,44 @@ export default class DateRangePicker extends React.Component {
     const { onFocusChange } = this.props;
     if (!this.isOpened()) return;
 
+    this.setState({
+      isDateRangePickerInputFocused: false,
+      isDayPickerFocused: false,
+      showKeyboardShortcuts: false,
+    });
+
     onFocusChange(null);
   }
 
+  onDateRangePickerInputFocus(focusedInput) {
+    const { onFocusChange, withPortal, withFullScreenPortal } = this.props;
+
+    if (focusedInput) {
+      const moveFocusToDayPicker = withPortal || withFullScreenPortal;
```

Comment:
hmm, well my thought is that it only makes sense to do this when the input is not visible... but maybe on touch devices makes sense too. Is the input going to be visible at the top of the VERTICAL_SCROLLABLE DayPicker? I forget what the designs look like